src/test/java/uk/gov/hmcts/pdda/business/services/publicdisplay/DisplayConfigurationHelperTest.java

```diff
@@ -0,0 +1,298 @@
+        } catch (Exception exception) {
+            fail(exception);
+        }
+        assertTrue(result, TRUE);
+    }
+
+    @Test
+    void testGetDisplayConfiguration() {
+        // Setup
+        XhbDisplayDao displayDao = DummyPublicDisplayUtil.getXhbDisplayDao();
+        EasyMock.expect(mockXhbDisplayRepository.findById(EasyMock.isA(Integer.class)))
+            .andReturn(Optional.of(displayDao));
+        EasyMock.replay(mockXhbDisplayRepository);
+        // Run
+        DisplayConfiguration result = classUnderTest.getDisplayConfiguration(0,
+            mockXhbDisplayRepository, mockXhbCourtRepository);
+        // Checks
+        EasyMock.verify(mockXhbDisplayRepository);
+        assertNotNull(result, NOT_NULL);
+    }
+
+    @Test
+    void testGetDisplayConfigurationNull() {
+        Assertions.assertThrows(DisplayNotFoundException.class, () -> {
+            classUnderTest.getDisplayConfiguration(0, mockEntityManager);
+        });
+    }
+
+    @Test
+    void testGetDisplayConfigurationMultiSite() {
+        // Setup
+        List<XhbCourtRoomDao> roomList = new ArrayList<>();
+        XhbCourtRoomDao xhbCourtRoomDao = DummyCourtUtil.getXhbCourtRoomDao();
+        xhbCourtRoomDao.setXhbCourtSite(DummyCourtUtil.getXhbCourtSiteDao());
+        roomList.add(xhbCourtRoomDao);
+
+        XhbDisplayDao xhbDisplayDao = DummyPublicDisplayUtil.getXhbDisplayDao();
+        xhbDisplayDao.setXhbCourtRooms(roomList);
+
+        XhbCourtDao xhbCourtDao = DummyCourtUtil.getXhbCourtDao(80, "TestCourt");
+        List<XhbCourtSiteDao> xhbCourtSites = new ArrayList<>();
+        xhbCourtSites.add(DummyCourtUtil.getXhbCourtSiteDao());
+        xhbCourtSites.add(DummyCourtUtil.getXhbCourtSiteDao());
+        xhbCourtDao.setXhbCourtSites(xhbCourtSites);
+
+        EasyMock.expect(mockXhbDisplayRepository.findById(EasyMock.isA(Integer.class)))
+            .andReturn(Optional.of(xhbDisplayDao));
+        EasyMock.expect(mockXhbCourtRepository.findById(EasyMock.isA(Integer.class)))
+            .andReturn(Optional.of(xhbCourtDao));
+
+        EasyMock.replay(mockXhbDisplayRepository);
+        EasyMock.replay(mockXhbCourtRepository);
+
+        // Run
+        DisplayConfiguration result = classUnderTest.getDisplayConfiguration(0,
+            mockXhbDisplayRepository, mockXhbCourtRepository);
+
+        // Checks
+        EasyMock.verify(mockXhbDisplayRepository);
+        EasyMock.verify(mockXhbCourtRepository);
+        assertNotNull(result, NOT_NULL);
+    }
+
+    @Test
+    void testUpdateDisplayConfiguration() {
+        // Setup
+        XhbRotationSetsDao xhbRotationSetsDao = DummyPublicDisplayUtil.getXhbRotationSetsDao();
+        XhbCourtRoomDao[] roomArray = {DummyCourtUtil.getXhbCourtRoomDao()};
+        XhbDisplayDao xhbDisplayDao = DummyPublicDisplayUtil.getXhbDisplayDao();
+
+        DisplayConfiguration displayConfiguration =
+            new DisplayConfiguration(xhbDisplayDao, xhbRotationSetsDao, roomArray);

```

Comment:
The setup up to this point in the test is repeated several times - please extract this into a method

Reply:
This setup refers to static methods so I can't see a benefit to extracting them. I could remove the variables completely although then there would be multiple calls to DummyPublicDisplayUtil.getXhbDisplayDao(); throughout the test which seems more long winded.